### PR TITLE
Feat/pnl updates

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Fix analytics table style bug
- Adds leverage points icon to dropdown
- Updates pnl data tables
- Fixes sNOTE holding bugs
- Fixes portfolio pending message spacing issues
- Replaces 0 with dashes

### CODE REVIEW

File: packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx [hunk 1]
```diff
@@ -140,11 +152,11 @@ export function useEarningsBreakdown(isGrouped: boolean) {
...
-        incentivesEarnings: totalIncentiveEarnings
-          ? totalIncentiveEarnings
-              .toFiat(baseCurrency)
-              .toDisplayStringWithSymbol(2, true, false, 'en-US', true)
-          : '0',
+        incentivesEarnings:
+          totalIncentiveEarnings &&
+          totalIncentiveEarnings.toFloat() > minEarningsValue
+            ? formatCryptoWithFiat(baseCurrency, totalIncentiveEarnings)
+            : '-',
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx]: I thought we were going to just show a zero in these cases? I don't think we should be doing this minEarningsValue comparison anywhere

File: packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx [hunk 2]
```diff
@@ -56,6 +60,7 @@ export function useEarningsBreakdown(isGrouped: boolean) {
   const baseCurrency = useFiat();
   const holdings = usePortfolioHoldings(network);
   const groupedHoldings = useGroupedHoldings(network) || [];
+  const minEarningsValue = 0.01;
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx]: I think this should be removed altogether

File: packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx [hunk 3]
```diff
@@ -201,11 +248,25 @@ export function useEarningsBreakdown(isGrouped: boolean) {
...
+          incentivesEarnings: {
+            data: [
+              {
+                displayValue:
+                  totalIncentiveEarnings &&
+                  totalIncentiveEarnings.toFloat() > minEarningsValue
+                    ? totalIncentiveEarnings
+                        ?.toFiat(baseCurrency)
+                        .toDisplayStringWithSymbol(2)
+                    : '-',
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx]: We should show zero on null

Comment (matthew-garrett) [reply packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx]: Teddy made a ticket to always show a dash https://app.asana.com/0/1205751432906245/1207540343338899

Lmk if you want me to change it back to 0

File: packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx [hunk 4]
```diff
@@ -238,6 +238,29 @@ export function useEarningsBreakdown(isGrouped: boolean) {
           isDebt
         );
 
+        const incentivesEarningsData =
+          b.tokenType === 'nToken'
+            ? totalIncentiveEarnings
+              ? totalIncentiveEarnings
+                  ?.toFiat(baseCurrency)
+                  .toDisplayStringWithSymbol(2)
+              : `${FiatSymbols[baseCurrency]} 0.00`
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-holdings/use-earnings-breakdown.tsx]: Is there supposed to be a space between the symbol and the zeros?

**Pull request merged**